src/components/theme/types.ts

```diff
@@ -1,4 +1,4 @@
-export type RealTheme = 'light' | 'light-hc' | 'dark' | 'dark-hc';
+export type RealTheme = string;
 export type ThemeType = 'light' | 'dark';
 
 export type Theme = 'system' | RealTheme;
```

Comment:
```suggestion
export type RealTheme = 'light' | 'light-hc' | 'dark' | 'dark-hc';
export type ThemeType = 'light' | 'dark';

export type Theme = 'system' | RealTheme  | string;
```

Reply:
That's not enough, problem is in RealTheme type itself